### DISCUSSION TITLE
It says on the site to build solutions you'll need NET 7 but does it work with NET 8?

### DISCUSSION THREAD

**Discussion by WWIA7062** [AssetRipper/AssetRipper#1366] [Q&A] [answered]
I was just asking because I found out that it said NET 7 was at End of Life and that I should get NET 8 instead but I just wanted to make sure that it'd work

**Comment by ds5678** [comment, marked as answer]
That's outdated. AssetRipper uses .NET 8 and will use .NET 9 at the official release in November or perhaps earlier depending on my anxiousness for C#13 features.
